Makefile

```diff
@@ -41,6 +41,7 @@ include network.env
 BASE_CMD = nice -n -20 ./build/bin/go-quai --$(NETWORK) --syncmode $(SYNCMODE) --verbosity $(VERBOSITY) --nonce $(NONCE)
 BASE_CMD += --http --http.vhosts=* --http.addr $(HTTP_ADDR) --http.api $(HTTP_API)
 BASE_CMD += --ws --ws.addr $(WS_ADDR) --ws.api $(WS_API)
+BASE_CMD += --slices $(SLICES)
```

Comment:
I think this meant as an alternative to the `REGION` and `ZONE` args we used to specify if running a slice. If so, please remove the now obsolete `REGION` and `ZONE` args